### DISCUSSION TITLE
All progress messages showing when doing a content transer/restore

### DISCUSSION THREAD

**Discussion by ghost** [umbraco/Umbraco-CMS#15673] [Features and ideas]
Today you "only" get one message at a time when content transfering, or restoring, such as "Review manifest on target" or "Create manifest on target"

Please do this:

![image](https://github.com/umbraco/Umbraco-CMS/assets/82879686/bc79f308-108b-4afb-8691-24a1f0ab5d6e)

I only included two messages here in my mock but the point should get across; let's make some use of all that blank space with each new log message so you can get a grip on how long each step take, when you started etc - such data is useful to have collected :)

Thank you!

**Comment by bergmania** [comment]
Hi @SOTFB.. Thanks for your suggestions.. 

This feature is not part of the CMS, so I'll close the discussion here.

To ensure the team working on Umbraco Deploy sees it, please create it here https://github.com/umbraco/Umbraco.Deploy.Issues/issues. Sadly I'm not able to transfer it.
